scripts/publish-staged

```diff
@@ -99,6 +99,8 @@ while getopts ":d:p:s:b:i:a:o:h" opt; do
 		d)
 			if [[ "${OPTARG}" = "false" ]]; then
 				DRYRUN=false
+			else
```

Comment:
This works well! Let's go ahead and do this to the other entrypoints that do the same flag handling to the other scripts too:

```
./scripts/repoint-latest
./scripts/stage
./scripts/ec2-stage
```

Reply:
we can remove stage / ec2 stage soon. Not sure about repoint.

Reply:
updated in all 3 other entrypoint scripts.